package.json

```diff
@@ -62,15 +62,38 @@
   },
   "size-limit": [
     {
-      "path": "dist/rtk-query.cjs.production.min.js",
-      "limit": "25 KB"
+      "name": "ESM full",
+      "path": "dist/esm/index.js",
+      "limit": "15 KB"
+    },
+    {
+      "name": "createApi + setupListeners",
+      "path": "dist/esm/index.js",
+      "import": "{ createApi, setupListeners }"
```

Comment:
Oh wow, I had no idea this was a thing!  That's really cool!

Reply:
I will, as soon as I recreate the changes I did here over there ;)